core/src/main/java/jenkins/model/NewViewLink.java

```diff
@@ -0,0 +1,52 @@
+package jenkins.model;
+
+import com.google.common.annotations.VisibleForTesting;
+import hudson.Extension;
+import hudson.model.Action;
+import hudson.model.TransientViewActionFactory;
+import hudson.model.View;
+import java.util.Collections;
+import java.util.List;
+
+@Extension
+public class NewViewLink extends TransientViewActionFactory {
+
+    @VisibleForTesting
+    static final String ICON_FILE_NAME = "folder";
+    @VisibleForTesting
+    public static final String URL_NAME = "newView";
+
+    @Override
+    public List<Action> createFor(View v) {
+        return Collections.singletonList(new Action() {
+
+            @Override
+            public String getIconFileName() {
+                if (!hasPermission(v)) {
+                    return null;
+                }
+
+                return ICON_FILE_NAME;
+            }
+
+            @Override
+            public String getDisplayName() {
+                if (!hasPermission(v)) {
+                    return null;
+                }
+
+                return Messages.NewViewLink_NewView();
+            }
+
+            @Override
+            public String getUrlName() {
+                return URL_NAME;
```

Comment:
There is also `ViewGroup` that can contain multiple views. And there nested view plugin that can nest vews into tabbar. Then potentially one page can have multiple new view buttons. Tabbar is already has `new view` that handle such situation correctly. 

Also new item is on top page, but new view is on bottom.